autonomy/replay/agent.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, agent_id: int, agent_data: Dict, registry_path: Path) -> None
             else:
                 self.agent_env[key] = value
 
-        self.agent_env[CONNECTION_ABCI_CONFIG_HOST] = "localhost"
```

Comment:
This should still work no?

Reply:
"localhost" tries to open an IPv6 connection, and then crashes.

Reply:
on the server side or the client side?